pkg/gitparse/gitparse.go

```diff
@@ -271,7 +273,10 @@ func (c *Parser) FromReader(ctx context.Context, stdOut io.Reader, commitChan ch
 		case isMinusDiffLine(line):
 			// NoOp. We only care about additions.
 		case isMessageLine(line):
-			currentCommit.Message.Write(line[4:])
+			if recentlyPassedAuthor {
+				currentCommit.Message.Write(line[4:])
+			}
+			recentlyPassedAuthor = false
```

Comment:
If `recentlyPassedAuthor` is set here, we can't have more than a single line for a commit message, and we could end up getting extra lines into the diff. I think putting it somewhere like `isIndexLine` (or something else that will come up after the commit message is done, but before a diff would start) should work though.

Reply:
Good call, `isIndexLine` should be good